srcpkgs/root/template

```diff
@@ -0,0 +1,94 @@
+# Template file for 'root'
+pkgname=root
+version=6.24.06
+revision=1
+# Only i686 and x86_64 seem to be officially supported
+archs="i686* x86_64*"
+build_style=cmake
+configure_args="-Dgnuinstall=ON -Druntime_cxxmodules=OFF -Dclad=OFF $(vopt_if fortran '-Dfortran=ON') -DXROOTD_XrdCl_LIBRARY=/usr/lib/libXrdCl.so.3 -DCMAKE_INSTALL_LIBDIR=/usr/lib/root -DCMAKE_INSTALL_SYSCONFDIR='../etc/root' -DCMAKE_INSTALL_PYTHONDIR=${py3_sitelib/'usr/'}"
+hostmakedepends="gcc-fortran"
+makedepends="libX11-devel libXpm-devel libXft-devel libXext-devel openssl-devel libpng-devel readline-devel tbb-devel liblz4-devel pcre-devel xxHash-devel libzstd-devel liblzma-devel libvdt-devel python3-devel python3-numpy"
+depends="gcc"
+case "$XBPS_TARGET_MACHINE" in
+	*-musl)
+	configure_args+=" -Dssl=OFF -Dxrootd=OFF"
+	;;
+	*)
+	makedepends+=" xrootd R-cran-Rcpp R-cran-RInside"
+	configure_args+=" -Dr=ON"
+	;;
+esac
+short_desc="C++ framework for data processing created by CERN"
+maintainer="Ben Jargowsky <benjar63@gmail.com>"
+license="LGPL-2.1-or-later"
+homepage="https://root.cern"
+distfiles="https://root.cern/download/root_v${version}.source.tar.gz"
+checksum=907f69f4baca1e4f30eeb4979598ca7599b6aa803ca046e80e25b6bbaa0ef522
+build_options="fortran"
+build_options_default=$build_options
+python_version=3
+shlib_provides="libCore.so libTree.so libRIO.so libMatrix.so libThread.so libTMVA.so libMathCore.so"
```

Comment:
common/shlibs is correct way to announce shared libs, no need for `shlib_provides=` in templates.

Reply:
Hm, when I didn't have that, it built right with xbps-src, but complained about missing, broken libraries when trying to install the subpackages with xbps-install. Maybe because those are in `/usr/lib/ROOT` and not in `/usr/lib`? I could perhaps move just those core libraries required by the subpackages into `/usr/lib`.

Reply:
Ok, symlinking the libraries needed to `/usr/lib` seems to solve this.

Reply:
You right, libraries go into package metadata only if exactly in `/usr/lib` or versioned.
Please keep shlib_provides= then.